docs/tutorial/Testing.md

```diff
@@ -34,6 +34,49 @@ test('reducer works as expected', (t) => {
 > not equal within JavaScript, and so are best to avoid if you want to compare
 > effects in your tests.
 
+## Working with nested cmd objects
+
+You may need to test objects from the `Cmd` module that make use of
+`Cmd.list` or `Cmd.map`. Imagine in the above example the reducer is updated to
+perform multiple commands when loading starts. We want to test that the properly
+shaped `fetchDetails` cmd is called, but don't necessarily want to mock out every
+effect in `Cmd.list` for an assertion. `flattenCmd` will return all commands nested in
+`Cmd.list`, `Cmd.map`, `Cmd.setInterval` and `Cmd.timeout` in a conveniently un-nested array that contains
+only `Cmd.run` and `Cmd.action` objects.
+
+```js
+import test from 'tape';
+import reducer, { fetchDetails } from './reducer';
+import { loadingStart, loadingSuccess, loadingError } from './actions';
+import { Cmd, loop, flattenCmd, getCmd } from 'redux-loop';
+
+test('reducer fetches data after loadingStart action', (t) => {
+  t.plan(2)
+  const state = { loading: false };
+
+  const resultCmd = getCmd(
+    reducer(state, loadingStart(1));
+  );
+
+  const fetchCmd = flattenCmd(resultCmd).find((cmd) => {
```

Comment:
it seems a little odd to search by successActionCreator only to then do a deep equal later. Why not just say

```ts
  expect(flattenCmd(resultCmd)).toContain(Cmd.run(fetchDetails, {
      successActionCreator: loadingSuccess,
      failActionCreator: loadingError,
      args: [1]
    }))
```

Reply:
This and `t.plan` where `tape` testing library specific. I've updated the examples to use the "jest" assertions which make the example tests a bit more clear